docs/CODE_GUIDELINES.md

```diff
@@ -622,22 +629,47 @@ class Foo
 };
 ```
 
-### 10.1. Destructors in interfaces
+### 11.5. Destructors in interfaces
 
 A class with any virtual functions should have a destructor that is either public and virtual or else protected and nonvirtual (cf. [ISO C++ guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-dtor-virtual)).
 
-## 10. Other conventions
+### 11.6. Constructor Initialzation Lists
+
+For short lines everything on one line is OK.
```

Comment:
The example that follows on the next lines has not "everything" on one line, namely the curly braces. How exactly do we want it? I'm in favor for the style used in the example (curly braces on extra lines).

Reply:
That's not possible as we use Allman braces. Will update description to match example.

Reply:
I guess I was not clear with my previous post. I'm all in for Allman. So, always having curly braces on next line is fine by me.

Reply:
Sorry, it QA I who misread!